lib/error.js

```diff
@@ -13,6 +13,12 @@ function FormError(key, options, req, res) {
             return options.message || this.getMessage(key, options, req, res);
         }
     });
+    Object.defineProperty(this, 'title', {
+        enumerable: true,
+        get: function () {
+            return options.title || 'Oops, something went wrong';
```

Comment:
(By the looks of it, getMessage just lets subclasses hook in and modify behaviour.)